src/hooks/useCustomWallpaper.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+
+const WALLPAPER_PATH = '/Photos/Settings/Wallpaper.jpg'
+
+const useCustomWallpaper = client => {
+  const [wallpaperLink, setWallpaperLink] = useState(null)
+  const [fetchStatus, setFetchStatus] = useState('idle')
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setFetchStatus('loading')
+        const response = await client
+          .collection('io.cozy.files')
+          .getDownloadLinkByPath(WALLPAPER_PATH)
```

Comment:
No way to have a realtime on this file ? :D

Reply:
Not a realtime download link no.

Reply:
It would be a cool upgrade, but definitely outside the scope of this PR.